.github/workflows/release.yml

```diff
@@ -80,6 +79,13 @@ jobs:
           pnpm changeset version --snapshot alpha
           pnpm build
           pnpm changeset publish --tag alpha
+
+          CURRENT_PACKAGE_VERSION=$(node -p "require('./packages/core/package.json').version")
```

Comment:
Ah thats a bit meh, but if it works we can clean it up later if we want to.

Reply:
Yeah it's a bit meh, they don't create a tag/commit when you do a snapshot (it's seems to be unusual to create tags for that)